src/ChangeStream.php

```diff
@@ -40,6 +40,8 @@ class ChangeStream implements Iterator
     private $csIt;
     private $key;
 
+    private $previousValid;
```

Comment:
Blank line above this can be removed so all `private` declarations are in the same paragraph.